src/classes/vocabulary/jsonld/class-post-jsonld.php

```diff
@@ -115,7 +154,75 @@ public function wl_after_get_jsonld( $jsonld ) {
 		}
 
 		return $jsonld;
-
 	}
 
+	/**
+	 * Set events request.
+	 *
+	 * @param $jsonld_arr array The final jsonld before outputting to page.
+	 * @param $post_id int The post id for which the jsonld is generated.
+	 * @param $context int A context for the JSON-LD generation, valid values in Jsonld_Context_Enum
+	 *
+	 * @return array
+	 */
+	public function set_events_request( $jsonld_arr, $post_id, $context ) {
+		// If context is not PAGE or the array is empty, return early.
+		if ( Jsonld_Context_Enum::PAGE !== $context || empty( $jsonld_arr[0] ) ) {
```

Comment:
If the array is empty, but before we sent a count of mentions and about then we should send an updated count of 0 for both.